driver-core/src/test/java/com/datastax/driver/core/EventDebouncerTest.java

```diff
@@ -65,6 +67,33 @@ public void should_deliver_single_event() throws InterruptedException {
     }
 
     @Test(groups = "unit")
+    public void should_log_and_drop_events_on_overflow() throws InterruptedException {
```

Comment:
Hmm are you sure we are not risking an OOM error here? Also, how long does this test take to complete?